src/components/TabsBar/styles.scss

```diff
@@ -0,0 +1,68 @@
+@import '../../utils/variables.scss';
+
+/* Dashboard Page */
+
+.dashboard {
+  display: flex;
+  justify-content: center;
+}
+
+.dashboard-card {
+  /* dashboard Card */
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+
+  width: 250px;
+  height: 90px;
+  box-shadow: $card-box-shadow;
+
+  background: $secondary-background;
+  border-radius: 16px;
+}
+
+.dashboard-margin {
+  margin: 2px;
+}
+
+.dashboard-text {
+  .ant-tabs-nav-list {
+    background-color: $secondary-background;
+    height: 50px;
+    width: 300px;
+    border-radius: 25px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+  .ant-tabs-tab-active .ant-tabs-tab-btn {
+    color: $black;
+    background-color: $secondary-color;
+    padding: 5px 25px;
+    border-radius: 25px;
+  }
+
+  .ant-tabs-ink-bar {
+    background: transparent;
+    pointer-events: none;
+  }
+
+  font-family: $raleway;
```

Comment:
for font family/typography, ideally we should do this when overriding antd theme.